### ISSUE TITLE
problem in startup of v8.0.4

### ISSUE THREAD

**Issue by herrJones** [Tribler/tribler#8270]
Hi! I was using Tribler and THIS happened! :cry:
```
Traceback (most recent call last):
  File "D:\a\tribler\tribler\src\tribler\core\restapi\rest_manager.py", line 110, in error_middleware
  File "D:\a\tribler\tribler\src\tribler\core\restapi\rest_manager.py", line 164, in required_components_middleware
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\aiohttp\web_middlewares.py", line 117, in impl
  File "D:\a\tribler\tribler\src\tribler\core\libtorrent\restapi\downloads_endpoint.py", line 330, in get_downloads
  File "D:\a\tribler\tribler\src\tribler\core\libtorrent\download_manager\download_manager.py", line 1121, in get_libtorrent_max_upload_rate
TypeError: '<' not supported between instances of 'int' and 'str'

```

**Comment by qstokkink** [comment]
Thanks for sharing! For some reason, your configuration file (`%APPDATA%/.Tribler/8.0/configuration.json`) got corrupted. There should be a line, in that file, that has the following key:

```
"max_upload_rate": 0,
```

In your case, this has apparently taken on a string value, something like:

```
"max_upload_rate": "0",
```

While we investigate what caused this, you have the following workarounds available:

 a. Remove the quotes around the value of `max_upload_rate`. For example, `0` instead of `"0"`.
 b. Remove the entire `configuration.json` to reset to default settings.

**Comment by herrJones** [comment]
OK, thanks, I'll try that.I upgraded from v8.0.2 to 8.0.4 and just migrated the dlcheckpoints.
I don't think I copied another file

    Op vrijdag 22 november 2024 om 11:13:11 CET schreef Dr. Quinten Stokkink ***@***.***>:  
 
 


Thanks for sharing! For some reason, your configuration file (%APPDATA%/.Tribler/8.0/configuration.json) got corrupted. There should be a line, in that file, that has the following key:
"max_upload_rate": 0,

In your case, this has apparently taken on a string value, something like:
"max_upload_rate": "0",

While we investigate what caused this, you have the following workarounds available:

a. Remove the quotes around the value of max_upload_rate. For example, 0 instead of "0".
b. Remove the entire configuration.json to reset to default settings.
